packages/ui/src/council/components/election/CandidateVote/CandidateVote.tsx

```diff
@@ -76,10 +77,17 @@ export const CandidateVote = ({
           <StakeAndVotesRow>
             {hasOwnStake && (
               <>
-                <Subscription>My Stake</Subscription>
+                <p data-for="tooltipForVotes"
+                   data-tip="Vote for your own membership with the same account as used for council candidacy creation counts towards the progress. <br/>
+                   Please note, this will in a voting lock applied to this account, which may only be withdrawn in the end of the council period <br/>
+                   if you win the election and released immediately if your candidacy gets outvoted by others.<br/>
+                   Voting locks are non-rivalrous.">
+                  <Subscription>My contributed votes</Subscription>
+                </p>
                 <StatsValue>
                   <TokenValue value={ownStake} />
                 </StatsValue>
+                <ReactTooltip id="tooltipForVotes" multiline={true} />
```

Comment:
You can use 
```
import { Tooltip, TooltipDefault } from '@/common/components/Tooltip'

         <Tooltip tooltipTitle="" tooltipText="" tooltipLinkURL="">
            <TooltipDefault />
          </Tooltip>
```

Reply:
Thanks for your suggestion.
I followed you and updated the code.

Reply:
The voting stake part looks good now. I think we can drop the `TooltipDefault` because it already has a child.

Reply:
I removed that line